surveys/2022-12/questions.md

```diff
@@ -0,0 +1,59 @@
+# Survey December 2022
+
+## Personal question
+
+- Which constituencies do you identify with?
+    - Direct end users: Users who run tools themselves
+    - Application operators: Users who interacts with existingm running applications
+    - Application developers: Frontend tools consumers, backend server authors, tools authors
+    - Library & package authors: Users who write librairies and packages to be included on other applications
+    - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+    - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+- Are you part of a constituency not covered ?
+    - yes
+    - no
+- If not covered what are you part with ? (open question)
+- Where are you from? (open question or countries list)
+    - country
+-  How long have you been working with Node.js? (numbers list)
+    - 0 - 10+
+- In what kind of organizations are you working in?
+    - big tech
+    - company
+    - startup
+    - university
+    - individual
+    - other
+- Are you part/following some Working Groups? 
+    - yes
+    - no
+- If part/following some Working Groups, which one? (open question)
+
+## Tech questions
+
+- What is important for you ? (Multiple choices)
+    - Good and up to date documentation
+    - Tutorial and examples of core / complexe features
+    - Internationalization of the documentation
+    - Good understanding of the direction of the project
+    - Predictable and stable release
+    - Innovation and new features 
+    - Consistent and intuitive error handling
+    - Runtime diagnostic tooling
+    - Relevant API in core
+    - Good understanding and process for security and CVE in the ecosystem
+    - Loader hook (to allow native typescript support)
```

Comment:
```suggestion
    - Hooks for customization, for example to use TypeScript without a separate build step
```

This isn’t specific to ESM.

Reply:
but the capability already exists in CJS.